src/membrane-keypad-element.ts

```diff
@@ -11,23 +20,33 @@ export class MembraneKeypadElement extends LitElement {
   @property() threeColumns = false;
 
   private pressedKeys = new Set<string>();
+  private shiftPressed = false;
 
   renderKey(text: string, x: number, y: number) {
     const keyClass = isNumeric(text) ? 'blue-key' : 'red-key';
+    let keyId = `key${text}`;
```

Comment:
Let's keep it consistent with the other elements on use kebab-case instead of camelCase (i.e. key-A instead of keyA)